tests/functional/src/test_sessions.py

```diff
@@ -1,8 +1,8 @@
 class TestSimple:
 
-    def test_session(self, selenoid_client):
+    def test_session(self, firefox_client):
         # get session_id
-        assert len(selenoid_client.sessions.id) > 10
+        assert len(firefox_client.sessions.id) > 10
```

Comment:
Magic number. Probably need to use `> 0`

Reply:
fixed in 9b20364a666f03413286df31a3d4c2cfb919f0c5